### PR TITLE
Formatting changes and version update

### PR DESCRIPTION
**Pull request by pcaronde**
Updated README and fixed version typo.

### CODE REVIEW

File: cs-deployment.yaml [hunk 1]
```diff
@@ -27,17 +27,17 @@ spec:
...
       containers:
       - image: empear/ubuntu-onprem
         env:
         - name: CODESCENE_DB_PATH
-          value: "/opt/codescene/codescene"
+          value: "/data/codescene/codescene"
         - name: CODESCENE_ANALYSIS_RESULTS_ROOT
-          value: "/opt/codescene/analysis"
+          value: "/data/codescene/analysis"
         - name: CODESCENE_CLOSED_REPOSITORIES_ROOT
```
Comment (jumarko) [inline cs-deployment.yaml]: This is a typo (probably) copy-pasted from the earlier version of https://github.com/empear-analytics/docker-codescene-nginx-self-signed-ssl.
Should be `CODESCENE_CLONED_REPOSITORIES_ROOT`.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # codescene-kubernetes
 ## Install Codescene standalone on kubernetes
-These instructions have been tested on AWS but should also work for GKE and standard K8S. The only *caveat* is configuring Persistent Storage is environment-specific.
+These instructions have been tested on AWS and GKE and standard K8S and with Istio. The only *caveat* is configuring Persistent Storage is environment-specific.
 
-##Memory allocation
+### Memory allocation
 In some docker installation documentation, the recommendation was for `300Mi`. Codescene 3.0, however, running on K8S seems to work better with a higher memory allocation. I have found that
```
Comment (jumarko) [inline README.md:6]: In what docker documentation they recommend 300 MB and how is it relevant to CodeScene?
Depending on the projects you analyze, CodeScene may require much more (a few gigabytes?) memory than just 500 MB.
Also, `I have found` should probably be `We have found`.

Comment (pcaronde) [reply README.md:6]: K8S allows you to define limits or requests. The first is a hard stop. The latter is a resource request which can, and usually does, go higher. I have defined resource requests not limits. If you prefer, you can change the lines to
`resources:
            requests:
              memory: "700Mi"
              cpu: "200m" `
 Use appropriate formatting.

File: cs-deployment.yaml (hunk 1 above)
Comment (pcaronde) [reply cs-deployment.yaml]: Indeed it is a typo. Good catch.

**Pull request merged**